addons/event/models/event_registration.py

```diff
@@ -249,6 +249,14 @@ def get_date_range_str(self):
         else:
             return _('on ') + format_datetime(self.env, self.event_begin_date, tz=self.event_id.date_tz, dt_format='medium')
 
-    def summary(self):
+    def registration_summary(self):
         self.ensure_one()
-        return {'information': []}
+        return {
+            'registration': {
+                'id': self.id,
+                'partner_id': self.partner_id.id,
+                'state': self.state,
+                'name': self.name or _('Attendee'),
+                'information': [(_('Event'), self.event_id.name), (_('Name'), self.name)]
```

Comment:
Could be simplified to return a dict (and not a dict within a "registration" key).

Also, the "information" key seems very random.
You already have the name (under the "name" key).
Could simply add a "event_name" key, and handle the rest at display ?